index.js

```diff
@@ -44,4 +39,45 @@ const nextOnNetlify = (options = {}) => {
   trackNextOnNetlifyFiles();
 };
 
+const watch = (functionsPath, publishPath) => {
+  logTitle(`👀 Watching source code for changes`);
+
+  const watcher = chokidar.watch(SRC_FILES);
+  const runBuild = debounceFn(
+    () => {
+      try {
+        execa.sync("next", ["build"], { stdio: "inherit" });
+        build(functionsPath, publishPath);
+      } catch (e) {
+        console.log(e);
+      }
+    },
+    {
+      wait: 3000,
+    }
+  );
+  watcher.on("add", runBuild);
```

Comment:
Would `watcher.on("all", runBuild)` work?

Reply:
Yes it will, thanks!